lib/awful/hotkeys_popup/widget.lua

```diff
@@ -371,6 +427,22 @@ function widget.new(args)
             beautiful.hotkeys_description_font or "Monospace 8"
         self.group_margin = args.group_margin or
             beautiful.hotkeys_group_margin or dpi(6)
+        self.highlight_bg = args.highlight_bg or
+            beautiful.hotkeys_highlight_bg or "#ffff00"
```

Comment:
please default to other `beautiful.*` variables which are for sure defined in the default theme.lua instead of falling back to hardcoded colors

Reply:
I like this bright yellow, because the point is to easily see highlighted text. For example firefox uses `#ff00ff`, also bright.

What about `beautiful.bg_urgent` (red)? I haven't found any other beautiful colors that fit it.

Reply:
yup, just choose any placeholder.
people who care more would set the right theme key anyway